docs/docs/quickstart.md

```diff
@@ -586,15 +618,15 @@ def run(plan, args):
     # ...
 
     postgres_url = "postgresql://{}:{}@{}:{}/{}".format(
-        "postgres",   # <---------- THE PROBLEM 
+        "postgres",   # <---------- THE PROBLEM IS HERE
         POSTGRES_PASSWORD,
         postgres.ip_address,
         postgres.ports[POSTGRES_PORT_ID].number,
         POSTGRES_DB,
     )
 ```
 
-In the line declaring the `postgres_url` variable in your `main.star` file, replace the `"postgres"` string with `POSTGRES_USER` to use the correct username we specified at the beginning of our file. Then rerun your dev loop:
+In the line declaring the `postgres_url` variable in your `main.star` file (line 58), replace the `"postgres",` string with `POSTGRES_USER,` to use the correct username we specified at the beginning of our file. Then re-run your dev loop:
```

Comment:
I'd caution against line numbers, because they're likely not going to get updated in the future.

Reply:
```suggestion
In the line declaring the `postgres_url` variable in your `main.star` file, replace the `"postgres",` string with `POSTGRES_USER,` to use the correct username we specified at the beginning of our file. Then re-run your dev loop:
```